docs/sdks/uid2-sdk-ref-ios.md

```diff
@@ -91,21 +100,31 @@ Install the iOS SDK via Swift Package Manager (SPM). There are two installation
 
 Add the following dependency to Package.swift:
 
-``` javascript
+```js
 dependencies: [
-.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "0.3.0"),
+.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "1.0.0"),
 ]
 ```
 
 ### Installing with Xcode
 
-Add the following dependency via Xcode:
+Add the following entry in the Package Dependencies for your apps: (**GWH__SW_18 is it correct re Package Dependencies, wording and capitalization?**)
+
+| Name | Location | Dependency Rule |
+| :--- | :--- |:-----------------------------| 
+| uid2-ios-sdk | git@github.com:IABTechLab/uid2-ios-sdk.git | Up to next major version: 1.0.0 < 2.0.0 |
 
-![Xcode Dependency](images/xcode-uid2-ios-sdk.png)
+<!-- ![Xcode Dependency](images/xcode-uid2-ios-sdk.png) --> 
+
+(**GWH__SW_19 should we remove this image? No longer needed? In which case we can remove this reference and also the actual file?**)
```

Comment:
Guess I missed that. Moved it now (but only in the English, not the Japanese, yet. Translator will have to do that because I can't update the content.